<!DOCTYPE html><html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><script type="text/javascript" src="https://file.bkxsj.com/skin/book/js/sk.js"></script><meta name="robots" content="index,follow"><title>Understanding enzymes: function[PDF|Epub|txt|kindle电子书版本网盘下载]-灵感之桥</title><meta name="Keywords" content="Understanding enzymes: function"/><meta name="description" content="Understanding enzymes: functionpdf下载文件大小为309MB,PDF页数为914页"/><meta http-equiv="X-UA-Compatible" content="IE=9; IE=8; IE=7; IE=EDGE;chrome=1"><link type="image/x-icon" rel="shortcut icon" href="https://www.shukui.net/skin/book/images/favicon.ico"><link type="text/css" rel="stylesheet" href="https://www.shukui.net/skin/book/css/style.css"><style>#main .d-main {margin-left: 0;width: 620px;}.down-btn {animation: myShake 2.5s linear .15s infinite}@keyframes myShake {0%, 66% {transform: translateZ(0)}67%, 73.6%, 83.6%, 93.6%, to {animation-timing-function: cubic-bezier(.215, .61, .355, 1);transform: translateZ(0)}80.3%, 81.4% {animation-timing-function: cubic-bezier(.755, .05, .855, .06);transform: translate3d(0, -4px, 0)}90.3% {animation-timing-function: cubic-bezier(.755, .05, .855, .06);transform: translate3d(0, -2px, 0)}97% {transform: translate3d(0, -.5px, 0)}}.copylink-btn {margin-right: 20px;}.copymd5-btn {margin-bottom: 25px;margin-left: 10px;}</style></head><body><div id="header"><div class="inner"><div class="logo"><a href="/"><img width="103" height="25" alt="灵感之桥"src="https://www.shukui.net/skin/book/images/logo.png"></a></div><div class="search"><form action="/so/search.php" target="_blank"><input type="text" autocomplete="off" id="bdcsMain" name="q" placeholder="书名 / 作者 / 出版社 / ISBN"class="inp-txt"><select class="inp-select" id="datasource" onchange="selectDatasource(this)"><option value="so">主库</option><option value="s">从库</option></select><input type="submit" value="搜索" class="inp-btn"></form></div></div></div><div id="main"><div class="d-main"><div class="tit"><h3>图书介绍</h3></div><h1 class="book-name">Understanding enzymes: functionPDF|Epub|txt|kindle电子书版本网盘下载</h1><div class="d-info"><div class="b-thumb"><img src="https://www.shukui.net/cover/22/31843019.jpg" alt="Understanding enzymes: function"></div><div class="b-info"><ul><li> 著</li><li>出版社： Pan Stanford Publishing Pte Ltd</li><li>ISBN：</li><li>出版时间：2016</li><li>标注页数：0页</li><li>文件大小：309MB</li><li>文件页数：914页</li><li>主题词：</li></ul></div></div><div class="tit"><h3>PDF下载</h3></div><div></br><a style="color:red;" rel="external nofollow" href="https://www.kjlm.net/ebook/1555827.html"target="_blank"><b>点此进入-本书在线PDF格式电子书下载【推荐-云解压-方便快捷】直接下载PDF格式图书。移动端-PC端通用</a></b></br><a class="down-btn" rel="external nofollow" href="https://down.trackerbk.com/bt/47/31843019.torrent"target="_blank">种子下载</a>[BT下载速度快]温馨提示：（请使用BT下载软件FDM进行下载）<a rel="nofollow" href="https://www.freedownloadmanager.org/zh/" target="_blank">软件下载地址页</a><a class="down-btn" rel="external nofollow" href="https://down.p2spdb.com/47/31843019.rar" target="_blank">直链下载</a>[便捷但速度慢]&nbsp;&nbsp;<a style="color:red;" rel="external nofollow" href="https://pdfyl.ertongbook.com/35/31843019.pdf" target="_blank"><b>[在线试读本书]</b></a>&nbsp;&nbsp;<b> <a style="color:red;" rel="external nofollow" href="https://web.jyjl.org/index/recovery.html" target="_blank">[在线获取解压码]</a></b><div class="copymd5-btn"><a href="javascript:copyToClip('609782cf8a344c4270d5d933639d4e69')">点击复制MD5值：609782cf8a344c4270d5d933639d4e69</a></div></div><div class="tit"><h3>下载说明</h3></div><div style="margin:20px 10px"><h2>Understanding enzymes: functionPDF格式电子书版下载</h2>下载的文件为RAR压缩包。需要使用解压软件进行解压得到PDF格式图书。<br><br><div class="copymd5-btn"><a href="javascript:copyToClip('magnet:?xt=urn:btih:RX6G6JQ2LJW7PBBEDNUSCQM7ICDMHIWK')">点击复制85GB完整离线版磁力链接到迅雷FDM等BT下载工具进行下载</a>&nbsp;&nbsp;<a rel="nofollow" target="_blank">详情点击-查看共享计划</a></div>建议使用BT下载工具Free Download Manager进行下载,简称FDM(免费,没有广告,支持多平台）。本站资源全部打包为BT种子。所以需要使用专业的BT下载软件进行下载。如BitComet qBittorrent uTorrent等BT下载工具。迅雷目前由于本站不是热门资源。不推荐使用！后期资源热门了。安装了迅雷也可以迅雷进行下载！<br><br><b>（文件页数 要大于 标注页数，上中下等多册电子书除外）</b><br><br><p style="color:red;"> <b>注意：本站所有压缩包均有解压码：</b> <a rel="nofollow" target="_blank"><b>点击下载压缩包解压工具</b></a></p></div><div class="tit"><h3>图书目录</h3></div><div id="book-contents"><p>PART Ⅰ ENZYME FUNCTION3</p><p>1 A Short Practical Guide to the Quantitative Analysis of Engineered Enzymes&amp;Christopher D.Bayer and Florian Hollfelder3</p><p>1.1 Introduction3</p><p>1.2 Quantifying Reaction Progress4</p><p>1.3 Typical Saturation Plots Give Michaelis-Menten Parameters5</p><p>1.4 What Can Go Wrong？8</p><p>1.5 Dealing with Multiphasic and Pre-Steady-State Kinetics12</p><p>1.6 Evaluating Enzymes16</p><p>2 Protein Conformational Motions：Enzyme Catalysis&amp;Xinyi Huang，C.Tony Liu，and Stephen J.Benkovic21</p><p>2.1 Introduction21</p><p>2.2 Multidimensional Protein Landscape and the Timescales of Motions22</p><p>2.3 Conformational Changes in Enzyme-Substrate Interactions26</p><p>2.4 Conformational Changes in Catalysis28</p><p>2.4.1 Protein Dynamics of DHFR in the Catalytic Cycle30</p><p>2.4.2 Temporally Overlap：Correlation Does Not Mean Causation32</p><p>2.4.3 Fast Timescale Conformational Fluctuations34</p><p>2.4.4 Effect of Conformational Changes on the Electrostatic Environment36</p><p>2.5 Conservation of Protein Motions in Evolution38</p><p>2.6 Designing Protein Dynamics39</p><p>2.7 Concluding Remarks40</p><p>3 Enzymology Meets Nanotechnology：Single-Molecule Methods for Observing Enzyme Kinetics in Real Time&amp;Kerstin G.Blank，Anna A.Wasiel，and Alan E.Rowan47</p><p>3.1 Introduction48</p><p>3.2 Single-Turnover Detection53</p><p>3.2.1 Fluorescent Reporter Systems53</p><p>3.2.2 Measurement Setup56</p><p>3.2.3 Data Analysis57</p><p>3.3 Single-Enzyme Kinetics60</p><p>3.3.1 Candida antarctica Lipase B63</p><p>3.3.2 Thermomyces lanuginosus Lipase67</p><p>3.3.3 α-Chymotrypsin73</p><p>3.3.4 Nitrite Reductase78</p><p>3.3.5 Summary84</p><p>3.4 New Developments Facilitated by Nanotechnology88</p><p>3.4.1 Nano-optical Approaches89</p><p>3.4.2 Nano-electronic Approaches96</p><p>3.4.3 Nanomechanical Approaches103</p><p>3.4.4 Summary108</p><p>3.5 Conclusion110</p><p>4 Interfacial Enzyme Function Visualized Using Neutron，X-Ray，and Light-Scattering Methods&amp;Hanna Wacklin and Tommy Nylander125</p><p>4.1 Phospholipase A2：An Interfacially Activated Enzyme126</p><p>4.1.1 Neutron Reflection129</p><p>4.1.2 Ellipsometry130</p><p>4.1.3 Activity of Naja mossambica mossambica PLA2130</p><p>4.1.4 Fate of the Reaction Products133</p><p>4.1.5 The Lag Phase and Activation of Pancreatic PLA2135</p><p>4.1.6 Distribution of Products during the Lag Phase138</p><p>4.1.7 Hydrolysis of DPPC by Pancreatic PLA2139</p><p>4.1.8 Role of the Reaction Products in PLA2 Activation141</p><p>4.1.9 Effect of pH and Activation by Me-β-cyclodextrin144</p><p>4.2 Other Lipolytic Enzyme Reactions on Surfaces150</p><p>4.2.1 Triacylglycerol Lipases and the Role of Lipid Liquid Crystalline Nanostructures150</p><p>4.3 Cellulase Enzymes154</p><p>4.4 Conclusion158</p><p>5 Folding Dynamics and Structural Basis of the Enzyme Mechanism of Ubiquitin C-Terminal Hydroylases&amp;Shang-Te Danny Hsu167</p><p>5.1 Introduction169</p><p>5.1.1 UCH-L1171</p><p>5.1.1.1 Genetic association between UCH-L1 and neurodegenerative diseases171</p><p>5.1.1.2 UCH-L1 in oncogenesis175</p><p>5.1.2 Molecular Insights into the Pathogenesis Associated with UCH-L1175</p><p>5.1.3 UCHL3177</p><p>5.1.4 UCHL5178</p><p>5.1.5 BAP1179</p><p>5.2 UCH Structures180</p><p>5.3 Folding Dynamics and Kinetics183</p><p>5.4 Substrate Recognition184</p><p>5.5 Enzyme Mechanism186</p><p>5.6 Conclusion189</p><p>6 Stabilization of Enzymes by Metal Binding：Structures of Two Alkalophilic Bacillus Subtilases and Analysis of the Second Metal-Binding Site of the Subtilase Family&amp;Jan Dohnalek，Katherine E.McAuley，Andrzej M.Brzozowski，Peter R.φstergaard，Allan Svendsen，and Keith S.Wilson203</p><p>6.1 Introduction：Subtilases and Metal Binding203</p><p>6.1.1 Calcium-Binding Sites in Bacillus：Proposal for a Standard Nomenclature209</p><p>6.1.2 The Weak Metal-Binding Site214</p><p>6.2 Two New Structures of Subtilases with Altered Calcium Sites216</p><p>6.2.1 Proteinase SubTY216</p><p>6.2.1.1 The overall fold216</p><p>6.2.1.2 The active site216</p><p>6.2.1.3 SubTY calcium and sodium sites218</p><p>6.2.1.4 SubTY disulfide bridge219</p><p>6.2.2 SubHal220</p><p>6.2.2.1 The unliganded form of SubHal220</p><p>6.2.2.2 The SubHal：CI2A complex221</p><p>6.2.2.3 Termini，surface，and pH stability of SubHal221</p><p>6.2.2.4 The two crystallographically independent SubHal：CI2A complexes223</p><p>6.2.2.5 The calcium sites in SubHal224</p><p>6.2.2.6 The active site of SubHal226</p><p>6.2.3 Enzymatic Activity of SubTY and SubHal228</p><p>6.2.4 Comparison of SubTY and SubHal with Other Subtilases228</p><p>6.2.5 The SubHal C-domain Compared to the Eukaryotic PCs，Furin and Kexin232</p><p>6.2.5.1 Active site comparison233</p><p>6.2.5.2 The specificity pockets234</p><p>6.2.5.3 Inhibitor CI2A binding234</p><p>6.2.6 Activity Profiles236</p><p>6.2.7 Comparison of Metal Binding at the Strong and Weak Sites in the S8 Family236</p><p>6.2.8 The Ca-Ⅱ and Na-Ⅱ Metal-Binding Sites237</p><p>6.3 Conclusion：Implications for Structural Studies of Enzymes248</p><p>6.4 Materials and Methods249</p><p>6.4.1 SubTY249</p><p>6.4.1.1 Protein production and purification249</p><p>6.4.1.2 Purification of the SubTY：CI2A （1：1）complex250</p><p>6.4.1.3 Crystallization250</p><p>6.4.1.4 Structure determination 2516.4.2 SubHal251</p><p>6.4.2.1 Protein production and purification251</p><p>6.4.2.2 Purification of the SubHal：CI2A （1：1）complex252</p><p>6.4.2.3 Crystallization252</p><p>6.4.2.4 Structure determination253</p><p>6.4.3 Protease Assays256</p><p>6.4.4 pH Stability257</p><p>6.4.5 Data Deposition257</p><p>7 Structure and Functional Roles of Surface Binding Sites in Amylolytic Enzymes&amp;Darrell Cockburn and Birte Svensson267</p><p>7.1 Introduction267</p><p>7.2 Identifiication of SBSs：X-Ray Crystallography271</p><p>7.3 Bioinformatics of SBS Enzymes273</p><p>7.4 Binding Site Isolation275</p><p>7.5 Protection of Binding Sites from Chemical Labeling277</p><p>7.6 Nuclear Magnetic Resonance277</p><p>7.7 Binding Assays278</p><p>7.8 Activity Assays282</p><p>7.9 Future Prospects283</p><p>7.10 Conclusion286</p><p>8 Interfacial Enzymes and Their Interactions with Surfaces：Molecular Simulation Studies&amp;Nathalie Willems，Mickael Lelimousin，Heidi Koldsφ，and Mark S.P.Sansom297</p><p>8.1 Introduction297</p><p>8.2 Enzyme Interactions at Interfaces299</p><p>8.3 Molecular Dynamic Simulations of Biomolecular Systems301</p><p>8.4 Lipases303</p><p>8.4.1 Atomistic MD Studies of Lipase Interactions with Interfaces304</p><p>8.4.2 The Role of Water in Lipase Catalysis at Interfaces307</p><p>8.5 Coarse-Grained MD Studies of Interfacial Enzymes：Orientation and Interactions309</p><p>8.5.1 Phospholipase A2309</p><p>8.5.2 PTEN310</p><p>8.6 Conclusions311</p><p>PART Ⅱ ENZYME DESIGN321</p><p>9 Sequence，Structure，Function：What We Learn from Analyzing Protein Families&amp;Michael Widmann and Jurgen Pleiss321</p><p>9.1 Introduction321</p><p>9.2 Detection of Inconsistencies Utilizing a Standard Numbering Scheme323</p><p>9.3 Identification of Functionally Relevant Positions327</p><p>9.4 The Modular Structure of Thiamine Diphosphate-Dependent Decarboxylases330</p><p>9.5 Stereoselectivity-Determining Positions：The S-Pocket Concept in Thiamine Diphosphate-Dependent Decarboxylases333</p><p>9.6 Regioselectivity-Determining Positions：Design of Smart Cytochrome P450 Monooxygenase Libraries336</p><p>9.7 Substrate Specificity-Determining Positions：The GX/GGGX Motif in Lipases340</p><p>9.8 Conclusion341</p><p>10 Bioinformatic Analysis of Protein Families to Select Function-Related Variable Positions&amp;Dmitry Suplatov，Evgeny Kirilin，and Vytas Svedas351</p><p>10.1 Introduction352</p><p>10.2 Bioinformatic Analysis of Evolutionary Information to Identify Function-Related Variable Positions359</p><p>10.2.1 Problem Definition359</p><p>10.2.2 Scoring Schemes in the Variable Position Selection：High-Entropy，Subfamily-Specific，and Co-Evolving Positions361</p><p>10.2.3 Association of the Variable Positions with Functional Subfamilies366</p><p>10.2.4 How to Select Functionally Important Positions as Hotspots for Further Evaluation：Implementation of Statistical Analysis366</p><p>10.3 The Bioinformatic Analysis of Diverse Protein Superfamilies369</p><p>10.3.1 Bioinformatic Challenges at Studying Enzymes369</p><p>10.3.2 Zebra：A New Algorithm to Select Functionally Important Subfamily-Specific Positions from Sequence and Structural Data370</p><p>10.4 Subfamily-Specific Positions as a Tool for Enzyme Engineering375</p><p>10.5 Conclusion377</p><p>11 Decoding Life Secrets in Sequences by Chemicals&amp;Zizhang Zhang387</p><p>11.1 Introduction388</p><p>11.2 Linking an Enzyme’s Activity to Its Sequence389</p><p>11.3 Refiining the Sequence Space to a Specifiic Function by Directed Evolution395</p><p>11.4 Linking Chemistry to -Omics with High-Throughput Screening Methods398</p><p>11.5 Finding Large Sequence Space of a Specific Function from Microbial Diversity400</p><p>11.6 Linking Sequences to Substromes at the Molecular Level404</p><p>11.6.1 Biocatalytic Study of EHs405</p><p>11.6.2 Pharmacological Study of EHs407</p><p>11.6.3 Mechanistic Study of EHs407</p><p>11.6.4 What We Have Learned from the Studies of EH410</p><p>11.6.5 Technologies with Potentials in Genochemistry Approach410</p><p>11.7 Correlating with Computational Methods410</p><p>11.8 Problems That Genochemistry Can Potentially Tackle413</p><p>11.9 Conclusion 41412 Role of Tunnels and Gates in Enzymatic Catalysis&amp;Sergio M.Marques，Jan Brezovsky，and Jiri Damborsky421</p><p>12.1 Introduction421</p><p>12.2 Protein Tunnels423</p><p>12.2.1 Structural Basis and Function423</p><p>12.2.2 Identification Methods427</p><p>12.2.3 Molecular Engineering429</p><p>12.3 Protein Gates431</p><p>12.3.1 Structural Basis and Function431</p><p>12.3.2 Identification Methods437</p><p>12.3.3 Molecular Engineering440</p><p>12.4 Conclusions442</p><p>13 Molecular Descriptors for the Structural Analysis of Enzyme Active Sites&amp;Valerio Ferrario，Lydia Siragusa，Cynthia Ebert，Gabriele Cruciani，and Lucia Gardossia465</p><p>13.1 Introduction：Molecular Descriptors for Investigation of Enzyme Catalysis465</p><p>13.2 Molecular Descriptors Based on Molecular Interaction Fields467</p><p>13.3 Multivariate Statistical Analysis for Processing and Interpretation of Molecular Descriptors472</p><p>13.4 Grind Descriptors for the Study of Substrate Specificity475</p><p>13.5 VolSurf Descriptors for the Modeling of Substrate Specifiicity477</p><p>13.6 Differential MIFs Descriptors for the Study of Enantioselectivity479</p><p>13.7 Hybrid MIFs Descriptors for the Computation of Entropic Contribution to Enantiodiscrimination481</p><p>13.8 Analysis of Enzyme Active Sites for Rational Enzyme Engineering484</p><p>13.9 BioGPS Descriptors for in silico Rational Design and Screening of Enzymes489</p><p>13.10 Conclusions495</p><p>14 Hydration Effects on Enzyme Properties in Nonaqueous Media Analyzed by MD Simulations&amp;Diana Lousa，Antonio M.Baptista，and Claudio M.Soares501</p><p>14.1 Enzyme Reactions in Nonaqueous Solvents502</p><p>14.2 Classes of Nonaqueous Solvents503</p><p>14.3 The Role of Water in Nonaqueous Biocatalysis504</p><p>14.4 Effect of Water Content on Enzyme Structure and Dynamics504</p><p>14.5 Effect of Water Content on Enzyme Selectivity507</p><p>14.6 Hydration Mechanisms of Enzymes in Polar and Nonpolar Solvents508</p><p>14.7 Enzyme Behavior as a Function of Water Activity510</p><p>14.8 Hydration Effects on Enzyme Reactions in Ionic Liquids512</p><p>14.9 Hydration Effects on Enzyme Reactions in Supercritical Fluids514</p><p>14.10 Conclusions516</p><p>15 Understanding Esterase and Amidase Reaction Specificities by Molecular Modeling&amp;Per-Olof Syren523</p><p>15.1 Introduction523</p><p>15.2 Fundamental Catalytic Concepts525</p><p>15.2.1 Fundamental Chemistry of Amides and Esters525</p><p>15.2.2 Esterases and Amidases and Their Metabolic Significance525</p><p>15.2.3 Fundamental Chemical Aspects of Amidase and Esterase Catalysis526</p><p>15.2.4 Impact of Stereoelectronic Effects on the Enzymatic Reaction Mechanism529</p><p>15.3 Molecular Modeling of Fundamental Catalytic Concepts529</p><p>15.3.1 QM Calculations on Amidases and Esterases529</p><p>15.3.2 MD Simulations on Amidases and Esterases535</p><p>15.3.3 QM/MM Simulations on Amidases and Esterases539</p><p>15.4 Outlook and Implications for Enzyme Design544</p><p>15.5 Additional Comments546</p><p>PART Ⅲ ENZYME DIVERSITY561</p><p>16 Toward New Nonnatural TIM-Barrel Enzymes Using Computational Design and Directed Evolution Approaches&amp;Mirja Krause and Rik K.Wierenga561</p><p>16.1 Introduction562</p><p>16.2 General Aspects of Protein Engineering566</p><p>16.2.1 Library Creation Methods569</p><p>16.2.2 Structure-Based Library Design572</p><p>16.2.3 Optimal Libraries for Directed Evolution Methods574</p><p>16.2.4 Data-Driven Design （Semirational Design）578</p><p>16.2.5 Protein Engineering by Selection and Screening Methods579</p><p>16.3 Directed Evolution Studies with TIM-Barrel Enzymes584</p><p>16.3.1 Protein Engineering Studies of TIM-Barrel Proteins586</p><p>16.3.2 The Kemp Eliminases590</p><p>16.4 Concluding Remarks596</p><p>17 Handling the Numbers Problem in Directed Evolution&amp;Carlos G.Acevedo-Rocha and Manfred T.Reetz613</p><p>17.1 Introduction614</p><p>17.2 Saturation Mutagenesis in Directed Evolution617</p><p>17.3 Statistical Analyses620</p><p>17.3.1 Conventional Statistics Based on the Patrick and Firth Algorithm620</p><p>17.3.2 Statistics Based on the Nov Algorithm624</p><p>17.4 How to Group and Randomize Amino Acid Positions626</p><p>17.5 Fitness Landscapes628</p><p>17.5.1 Fujiyama vs.Badlands Fitness Landscapes628</p><p>17.5.2 Fitness-Pathway Landscapes and How to Escape from Local Minima630</p><p>17.6 Conclusions and Perspectives636</p><p>18 Hints from Nature：Metagenomics in Enzyme Engineering&amp;Esther Gabor，Birgit Heinze，and Jurgen Eck643</p><p>18.1 Metagenomics and the Ideal Enzyme644</p><p>18.2 Molecular Microdiversity647</p><p>18.3 Metagenomic Enzyme Chimera650</p><p>18.4 Outlook653</p><p>19 A Functional and Structural Assessment of Circularly Permuted Bacillus circulans Xylanase and Candida antarctica Lipase B&amp;Stephan Reitinger and Ying Yu657</p><p>19.1 Introduction657</p><p>19.2 Naturally Occurring Circular Permutations：Selected Examples658</p><p>19.3 Circular Permutation of Bacillus circulans Xylanase661</p><p>19.4 Circular Permutation on Candida antarctica Lipase B669</p><p>19.5 Conclusion674</p><p>20 Ancestral Reconstruction of Enzymes&amp;Satoshi Akanuma and Akihiko Yamagishi683</p><p>20.1 Introduction683</p><p>20.2 Reconstruction of an Ancestral Protein Sequence684</p><p>20.2.1 Overview684</p><p>20.2.2 Methods for Ancestral Sequence Reconstruction684</p><p>20.2.3 Early Works686</p><p>20.3 The Commonote687</p><p>20.3.1 The Last Universal Common Ancestor，the Commonote687</p><p>20.3.2 Theoretical Studies on the Environmental Temperature of the Commonote688</p><p>20.3.3 Reconstruction of an Ancestral Nucleoside Diphosphate Kinase689</p><p>20.3.4 Estimation of the Environmental Temperature of the Commonote692</p><p>20.4 Application to Designing Thermally Stable Proteins693</p><p>20.4.1 Design of Thermally Stable Proteins693</p><p>20.4.2 Case Studies to Create Thermally Stable Enzymes by Introducing Ancestral Residues as Amino Acid Substitutions694</p><p>20.4.3 Reconstruction of Thermally Stable，Ancestral DNA Gyrase Using a Small Set of Homologous Amino Acid Sequences696</p><p>20.5 Conclusion697</p><p>PART Ⅳ ENZYME SCREENING AND ANALYSIS707</p><p>21 High-Throughput Screening or Selection Methods for Evolutionary Enzyme Engineering&amp;Shuobo Shi，Hongfang Zhang，Ee Lui Ang，and Huimin Zhao707</p><p>21.1 Introduction708</p><p>21.2 Selection710</p><p>21.2.1 Solid-Medium-Based Selection717</p><p>21.2.2 Liquid-Medium-Based Selection719</p><p>21.2.3 Display-Based Selection722</p><p>21.3 Screening724</p><p>21.3.1 Chromatography- and Mass-Spectrometry-Based Screening725</p><p>21.3.2 Solid-Medium-Based Screening726</p><p>21.3.3 Microtiter-Plate-Based Screening727</p><p>21.3.4 Yeast Two-/Three-Hybrid System729</p><p>21.3.5 FACS-Based Screening729</p><p>21.3.6 Microfluidics-Based Screening732</p><p>21.4 Conclusions and Prospects734</p><p>22 Nanoscale Enzyme Screening Technologies&amp;Helen Webb-Thomasen and Andreas H.Kunding745</p><p>22.1 Introduction745</p><p>22.2 Approaches to Nanocompartmentalization of Enzymes746</p><p>22.2.1 Liposomes747</p><p>22.2.1.1 Addressability747</p><p>22.2.1.2 Reagent exchange749</p><p>22.2.2 Polymersomes and VirusLike Particles751</p><p>22.2.3 Water-in-Oil Emulsion Droplets752</p><p>22.2.3.1 Addressability755</p><p>22.2.3.2 Reagent exchange755</p><p>22.3 Microfabricated Chip Devices for Enzyme Compartmentalization and Screening756</p><p>22.3.1 Microfluidic-Generated Emulsion Droplets757</p><p>22.3.2 Microfabricated Arrays762</p><p>22.3.2.1 Optical fiber microarrays762</p><p>22.3.2.2 Elastomeric microarrays763</p><p>22.3.2.3 Surface tension microarrays765</p><p>22.4 Conclusion and Current Challenges767</p><p>22.5 Future Improvements769</p><p>23 Computational Enzyme Engineering：Activity Screening Using Quantum Chemistry&amp;Martin R.Hediger777</p><p>23.1 Motivation778</p><p>23.2 Introduction779</p><p>23.3 Methods780</p><p>23.3.1 Calculation Engines780</p><p>23.3.2 Molecular Modeling782</p><p>23.3.3 Software786</p><p>23.4 Applications786</p><p>23.4.1 Overview786</p><p>23.4.2 Engineering Candida antarctica Lipase B787</p><p>23.4.3 Engineering Bacillus circulans Xylanase793</p><p>23.5 Conclusions800</p><p>24 In Silico Screening of Enzyme Variants by Molecular Dynamics Simulation&amp;Hein J.Wijma805</p><p>24.1 Potential Applications of MD Simulations For Improving Enzymes805</p><p>24.2 Molecular Dynamics vs.Other in silico Methods809</p><p>24.3 Improving Catalytic Activity by MD Screening812</p><p>24.3.1 Transition-State Simulation812</p><p>24.3.2 High-Energy Intermediate Simulation814</p><p>24.3.3 Substrate Simulation with Near-Attack Conformations815</p><p>24.3.4 Substrate Simulation with Monitoring of H Bonds817</p><p>24.4 Predicting and Improving Binding Affiinity818</p><p>24.5 MD Screening to Improve Enzyme Stability819</p><p>24.6 Improving Correlation between MD and Experiment822</p><p>24.6.1 Force Field Inaccuracies822</p><p>24.6.2 Sampling Concerns823</p><p>24.6.3 Other Concerns824</p><p>24.7 Outlook and Further Possibilities825</p><p>25 Kinetic Stability of Variant Enzymes&amp;Jose M.Sanchez-Ruiz835</p><p>25.1 Kinetics vs&amp;Thermodynamics in Protein Stability835</p><p>25.2 Mutation Effects on Kinetic Stability：A Description Based on the Transition State for Irreversible Denaturation838</p><p>25.3 Kinetic Stability Linked to the Breakup of Interactions in the Transition State：Pro-dependent Proteases841</p><p>25.4 Kinetic Stability Linked to Substantially Unfolded Transition States：Thioredoxin and Phytase Enzymes842</p><p>25.5 Role of Solvation Barriers in Kinetic Stability：Lipases and Triose Phosphate Isomerases848</p><p>25.6 Concluding Remarks852</p><p>Index859</p><p></p></div></div><div class="d-rt"><h3>热门推荐</h3><ul><li><a href="/book/2794546.html">2794546.html</a></li><li><a href="/book/1720996.html">1720996.html</a></li><li><a href="/book/429448.html">429448.html</a></li><li><a href="/book/3745152.html">3745152.html</a></li><li><a href="/book/2433177.html">2433177.html</a></li><li><a href="/book/3461540.html">3461540.html</a></li><li><a href="/book/905213.html">905213.html</a></li><li><a href="/book/2161483.html">2161483.html</a></li><li><a href="/book/3659023.html">3659023.html</a></li><li><a href="/book/814580.html">814580.html</a></li></ul></div></div><div id="footer"><p>Copyright&nbsp;&copy;&nbsp;2025&nbsp;&nbsp;<a href="/list/">最新更新</a></p><p>请使用FDM BitComet qBittorrent uTorrent等BT下载工具，下载本站电子书资源！首推Free Download Manager下载软件。文件页数>标注页数[分册图书除外]</p></div></body></html>